src/Models/Individuals/SugarKelp/coupling.jl

```diff
@@ -1,19 +1,18 @@
 #:NO₃, :NH₄, :DIC, :O₂, :DOC, :DON, :bPOC, :bPON
 
-@inline function (kelp::SugarKelp)(::Val{:NO₃}, t, A, N, C, u, v, w, T, NO₃, NH₄, PAR)
+@inline function (kelp::SugarKelp)(::Val{:NO₃}, t, A, N, C, u, v, w, T, NO₃, NH₄, PAR, P)
```

Comment:
```suggestion
@inline function (kelp::SugarKelp)(::Val{:NO₃}, t, A, N, C, u, v, w, T, NO₃, NH₄, PAR, photosynthesis)
```

I in this context its a bit confusing to use `P` since `P` almost always refers to phytoplankton